src/components/inputs/LCheckboxList.vue

```diff
@@ -1,14 +1,25 @@
 <template>
   <div class="LCheckboxList">
     <div class="LCheckboxList__header">
-      <l-input-solo
+      <l-text-field 
         v-model="inputSearch"
         :placeholder="translation.input || $t('ayla.search')"
-        append-icon="mdi-magnify"
+        class="LCheckboxList__input"      
```

Comment:
```suggestion
        class="LCheckboxList__input" 
```
Só remove esse espaço aqui que pegou a mais